docs/components/query.md

```diff
@@ -460,3 +469,13 @@ Flags:
                                  Prometheus.
 
 ```
+
+## Exported metrics
+
+Thanos Query also exports metrics about its own performance. You can find a list with these metrics below.
+
+**Disclaimer**: this list is incomplete. The remaining metrics will be added over time.
```

Comment:
May I request adding an issue for this? :innocent: I think this would be good first issue for someone.

Reply:
Done: #5758.